app/views/wikis/_form.html.erb

```diff
@@ -1,7 +1,7 @@
-<% if current_user.admin? || current_user.premium? %>
+<% if user_admin_or_premium? %>
   <div class="form-group">
     <%= f.label :private, class: 'checkbox' do %>
-      <%= f.check_box :private %> Private Wiki
+      <%= f.check_box :private, :true %> Private Wiki?
     <% end %>
```

Comment:
@bjblock - I've been working on this today and I'm still getting an undefined method of `|f|` here. This partial is called within the do block in the view where `|f|` is used as a placeholder so why am I getting an undefined method on `|f|`? I browsed the web and found that I should be defining `f` within the locals. I tried that but I didn't have much luck either. Any hints? I feel like the do block at the beginning of the view is ending before we get to this partial, is that correct?